txpipe/twopoint.py

```diff
@@ -1028,17 +1288,18 @@ def load_lens_catalog(self, data):
         print(f"Loading lens sample from {filename}")
 
         f = self.open_input('star_catalog')
-        
+
         mags = f['stars/r_mag'][:]
         bright_cut = mags>14
         bright_cut &= mags<18.3
-        
+
         data['lens_ra']  = f['stars/ra'][:][bright_cut]
         data['lens_dec'] = f['stars/dec'][:][bright_cut]
         f.close()
 
         npoint = data['lens_ra'].size
         data['lens_bin'] = np.zeros(npoint)
+        print('len lens', len(data['lens_ra']))
```

Comment:
is this needed?  If so make it clearer output.

Reply:
I just removed that line, if it is still needed feel free to add it back